src/indexer.c

```diff
@@ -1119,6 +1119,9 @@ void git_indexer_free(git_indexer *idx)
 	if (idx == NULL)
 		return;
 
+	if (idx->have_stream)
+		git_packfile_stream_free(&idx->stream);
```

Comment:
Meh, this function doesn't conform to our usual style. 'free' would usually denote the structure itself bein free'd as well, which would have segfaulted here. But it only calls inflateEnd, so this looks good.